packages/apify/src/storages/request_queue.ts

```diff
@@ -676,9 +669,10 @@ export class RequestQueue {
      *   ID or name of the request queue to be opened. If `null` or `undefined`,
      *   the function returns the default request queue associated with the actor run.
      * @param [options]
-     * @param [options.forceCloud=false]
+     * @param [options.forceCloud]
      *   If set to `true` then the function uses cloud storage usage even if the `APIFY_LOCAL_STORAGE_DIR`
      *   environment variable is set. This way it is possible to combine local and cloud storage.
+     * @default false
      */
     static async open(queueIdOrName?: string | null, options: { forceCloud?: boolean } = {}): Promise<RequestQueue> {
```

Comment:
let's make this an interface too, i don't think the API docs can handle the `options.forceCloud` jsdoc